modules/core/src/v2/coins/sol.ts

```diff
@@ -83,11 +85,32 @@ export class Sol extends BaseCoin {
   }
 
   verifyTransaction(params: VerifyTransactionOptions, callback?: NodeCallback<boolean>): Bluebird<boolean> {
-    throw new MethodNotImplementedError('verifyTransaction method not implemented');
+    return co<boolean>(function* () {
```

Comment:
lets implement `verifyTransaction` in a separate PR to keep this change more focused.

there are a lot more checks that are needed.
namely, we need to verify that the instructions when deserializing `txPrebuild.txBase64` matches the info from `params.txInfo`

Reply:
I create [this story](https://bitgoinc.atlassian.net/browse/STLX-10202) to implement verifyTransaction.
Could you please add a description and a DoD? (I think it will be better if you add it)